rss2email/feed.py

```diff
@@ -1011,8 +1021,12 @@ def _send_digest(self, digest, sender):
         any messages in the digest, don't call this function.
         """
         digest['From'] = sender
-        last_part = digest.get_payload()[-1]
-        last_message = last_part.get_payload()[0]
+        if (self.digest_type == 'multipart/digest'):
```

Comment:
More parentheses.

Reply:
Should have realized it.  They were copies of each other.